_CoqProject

```diff
@@ -0,0 +1,3 @@
+-arg -boot
```

Comment:
I think this is wrong.  I think some of the files rely on `Coq.Init.Prelude` being loaded by default, and rely on `-I plugins` (which I think follows from `-coqlib .`).  Does PG parse `-coqlib` in `_CoqProject`?

Reply:
In fact `-boot` implies the right `-coqlib`, see https://github.com/coq/coq/blob/master/lib/envars.ml#L121-L124.
The other uses are camlp4(5) detection also in envars.ml, adding stm/ and ide/ to loadpath in https://github.com/coq/coq/blob/master/toplevel/coqinit.ml#L89-L92 and whatever https://github.com/coq/coq/blob/master/tools/coqmktop.ml#L276 does.

I tried a few random files and they worked.

PG doesn't parse coqlib.

Reply:
Fascinating.

https://github.com/coq/coq/blob/master/tools/coqmktop.ml#L276 is responsible for adding the correct directories to the OCaml loadpath via `-I` when running `coqtop` from an installed location rather than from a local location.  I believe `-boot` has the same effect as `-coqlib .` in this case.

Reply:
@JasonGross do you still want it changed or should I dismiss review?

Reply:
I'm fine with leaving this.  I think the double dot on configure should be changed, and I plan to change my review to accept once that's updated.  (Though I'll be basically off the grid for a week, so feel free to dismiss my review if you fix that.)